rocky/reports/views/base.py

```diff
@@ -28,8 +28,11 @@ class ReportType(TypedDict):
 class ReportBreadcrumbs(OrganizationView, BreadcrumbsMixin):
     current_step: int = 1
 
-    def get_selection(self):
-        return "?" + urlencode(self.request.GET, True)
+    def get_selection(self, pre_selection: Optional[Dict[str, Union[str, List[str]]]] = None) -> str:
+        selection = self.request.GET.copy().dict()
+        if pre_selection:
+            selection.update(pre_selection)
```

Comment:
I think this will end up always overwriting the selection with the pre_selection, which is not what we want I suppose since it should act as a default when there is no selection